go-controller/pkg/util/node_annotations.go

```diff
@@ -235,9 +235,15 @@ func ParseNodeManagementPortMACAddress(node *kapi.Node) (net.HardwareAddr, error
 	return net.ParseMAC(macAddress)
 }
 
-func SetNodePrimaryIP(nodeAnnotator kube.Annotator, protocol iptables.Protocol, nodeIP string) error {
-	ipFamily := getFamilyFromProtocol(protocol)
-	return nodeAnnotator.Set(fmt.Sprintf("%s-%s", ovnNodeCIDR, ipFamily), nodeIP)
+func SetNodePrimaryIP(nodeAnnotator kube.Annotator, nodeIPNetv4, nodeIPNetv6 *net.IPNet) error {
+	var err error
+	if nodeIPNetv4 != nil {
+		err = nodeAnnotator.Set(fmt.Sprintf("%s-%s", ovnNodeCIDR, "ipv4"), nodeIPNetv4.String())
```

Comment:
It would be more consistent with the rest of ovn-kubernetes to have a single array-valued annotation rather than separate string-valued ipv4 and ipv6 annotations.

If you are going to keep the separate annotations, it would be simpler at this point to have separate `ovnNodeCIDRIPv4` and `ovnNodeCIDRIPv6` constants